client/components/common/TestPlanResultsTable/index.jsx

```diff
@@ -89,9 +89,9 @@ const TestPlanResultsTable = ({
                     {
                         id: `UnexpectedBehavior_MUST_${nextId()}`,
                         assertion: {
-                            text: 'Other behaviors that create high negative-impacts are not exhibited'
+                            text: 'Other behaviors that create severe negative-impacts are not exhibited'
```

Comment:
The hyphen here isn't grammatically correct, it should be `'Other behaviors that create severe negative impacts are not exhibited'`.

Reply:
True, although unsure if it was intentional. This could also be [shortened in the future soon](https://github.com/w3c/aria-at/issues/1043) to: 

> Severe negative side effects do not occur

If you'd like to weigh in on that discussion.